modules/docs/assets/validation_class_diagram.svg

```diff
@@ -0,0 +1 @@
3 C154.125,152.7813 154.8125,152.6563 155.2344,152.3906 C155.4219,152.2969 155.4219,152.2969 155.875,151.8125 C156.0625,151.625 156.2656,151.5469 156.4844,151.5469 C156.9531,151.5469 157.3438,151.9375 157.3438,152.3906 C157.3438,152.7813 157.0156,153.2344 156.4375,153.6406 C155.6875,154.1875 154.5313,154.4844 153.3125,154.4844 C150.4219,154.4844 148.3125,152.5938 148.3125,150.0156 L148.3125,148.8906 C148.3125,146.1719 150.3125,144.0938 152.9375,144.0938 C153.8125,144.0938 154.4063,144.2344 155.3438,144.6719 Z "/><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="115" x="167" y="153.1543">schema_validation</text><line style="stroke: #A80036; stroke-width: 1.5;" x1="139" x2="284" y1="165" y2="165"/><line style="stroke: #A80036; stroke-width: 1.5;" x1="139" x2="284" y1="173" y2="173"/><path d="M129.196,56.135 C113.775,76.751 91.1923,106.944 74.8979,128.729 " fill="none" style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 7.0,7.0;"/><polygon fill="#A80036" points="71.7269,132.968,80.3214,128.1581,74.7223,128.9646,73.9159,123.3655,71.7269,132.968" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="19" x="107.5" y="99.0669">init</text><path d="M164.86,67.742 C175.894,88.622 189.63,114.614 199.329,132.968 " fill="none" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#FFFFFF" points="158.726,56.135,157.9928,63.3087,164.3328,66.7446,165.0659,59.5709,158.726,56.135" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="8" x="152.7311" y="75.9291">1</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="24" x="160.1102" y="121.9718">0..2</text><path d="M65.5974,181.135 C75.4452,201.666 89.8485,231.6926 100.2879,253.4561 " fill="none" style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 7.0,7.0;"/><polygon fill="#A80036" points="102.4523,257.9684,102.1663,248.1237,100.2898,253.4602,94.9532,251.5837,102.4523,257.9684" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="58" x="89.5" y="224.0669">generate</text><path d="M185.241,191.639 C176.175,203.196 165.927,216.1931 156.5,228 C148.663,237.815 139.944,248.5887 132.417,257.8444 " fill="none" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#FFFFFF" points="193.507,181.086,186.6583,183.3432,186.1077,190.5332,192.9564,188.2761,193.507,181.086" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="8" x="170.8463" y="201.2638">1</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="8" x="128.2395" y="246.8166">1</text></g></svg>
```

Comment:
@pestOO is there some plugin that can generate duxygen documentation with usage of plantuml?